src/components/common/SelectPhotos.tsx

```diff
@@ -72,7 +73,7 @@ export default function SelectPhotos(props: SelectPhotosProps): JSX.Element {
     }
 
     if (newFiles.length) {
-      updateSelection([...files, ...newFiles]);
+      updateSelection([...files, ...newFiles].slice(0, maxPhotos));
```

Comment:
Some thoughts here:
1. show the max count allowed with some messaging in this widget (if a max is set)?
2. disable the photo selector button if max was reached?